### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by johnkioko**
**### You should implement the following interactions:**

1.Implemented popup window that pops when the user clicks (or taps) the button to check project details, the with details about the project appears.

2. Implemented the close (X) button such that when the user clicks (or taps) the popup disappears.
In order to associate each project with the popup window details, you must refactor the project section:

### CODE REVIEW

File: popup.css [hunk 1]
```diff
@@ -0,0 +1,922 @@
...
+  }
+
+  .recent-desk {
+    display: none;
+  }
+
+  .projects > hr {
+    width: 300px;
+  }
+
```
Comment (RolandM99) [inline popup.css:318]: - [ ] Kindly, make sure that your design for the popup window for the mobile version looks exactly like the original design from Figma.
take a look at the table below:

|Figma Design|Your Design|
|---------------|-------------|
|image|image|

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi  @johnkioko , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -64,32 +74,44 @@ <h2 class="header-section-heading-2">Software Developer</h2>
...
+        <h3>Skills</h3>
+        <ul>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>GitLab</li>
+          <li>Terminal</li>
+        </ul>
+      </article>
+    </section>
```
Comment (rotimiazeez) [inline index.html:113]: - Please note that the icons on your page are broken, kindly fix them. 👍🏽
Screenshot from 2022-09-09 22-45-13
Screenshot from 2022-09-09 22-44-11

Comment (rotimiazeez) [reply index.html:113]: - Please be sure your design matches the original Figma design for a better User Interface(UI) 😉.
Your Design | Figma Design
Screenshot from 2022-09-09 22-45-13 | Screenshot from 2022-09-09 22-48-12

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @johnkioko 👍🏽 , 
## You're almost done here 🤏🏽 
### Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @johnkioko , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive PR.
:heavy_check_mark: Well implemented suggested changes.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "Fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**